### DISCUSSION TITLE
Weird behavior: Nameservers method verification does not show on some domain

### DISCUSSION THREAD

**Discussion by charlie-ttt** [vercel/community#6763] [Help]
### Summary

I'm facing an issuing of not able to see the Nameservers method domain verification. I'm trying to set up a wildcard domain name and need to use this method.

As can see from the attached image, I tried adding two domains; but only one of them allow the nameservers method verification. Why is this? 

![image](https://github.com/vercel/community/assets/38147717/cd33daa8-aee9-4b56-9a18-6a7980119495)

Another weird thing is: even if it's not shown, I can verify it using the `ns1.vercel-dns.com` `ns2.vercel-dns.com`. It would then say Valid Configuration on the project's domain configuration page https://vercel.com/kop-bda4e78a/test/settings/domains
BUT, it doesn't show up on my team setting https://vercel.com/kop-bda4e78a/~/domains  which I require to manage my dns settings

### Steps to Reproduce

1. Add domains to project
2. Verify the TXT DNS verification on GoDaddy
3. See either 1 or 2 ways of dns verifications  (Expected to see 2 options for all domains)

**Comment by charlie-ttt** [comment]
Hi @amyegan Could you help guide me on this issue? I'm running out of ideas. 

I feel like it is a straightforward issue that doesn't have documentation on - what is preventing me from adding a new domain using Nameserver method?

The accepted answer on [here](https://github.com/orgs/vercel/discussions/6192#discussioncomment-9330398) doesn't seems like an official fix or solution (fixed by just transferring the domain around to different project).

**Comment by amyegan** [comment]
Hey @charlie-ttt. If you the domain was already owned by another account, you wouldn't be able to use nameserver configuration since you would need to include a TXT recored to [verify domain access](https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access). 

That's essentially what was happening in the other post which is why transferring the domain solved it in their case.

**Reply by charlie-ttt** [reply]
Thanks for clarifying. For my case though, it is a new domain not owned by another account.   I'm trying multiple times but still not able to see the Nameservers method showing up.   I'm trying to set up wildcard domain for my app website but this issue is preventing me.  Could you give me some advice or let me know what other details you need from me?

![image](https://github.com/vercel/community/assets/38147717/7f30d3c5-f0a4-4e7b-9976-175a83c02ffa)


*_Update_
I tried just setting the nameservers of my godaddy to `ns1.vercel-dns.com` and `ns2.vercel-dns.com` (even though the nameserver method doesn't show on the UI) and the domains now showed up as verified. **HOWEVER**, I cannot configure my DNS from Vercel because it doesn't show up on my domain pages... both on https://vercel.com/charliettts-projects/~/domains and https://vercel.com/account/domains. I need to do this because I need to configure my email domains in there. 

Could you give advice on how to set up the Nameservers method properly and how to enable Vercel DNS?

![image](https://github.com/vercel/community/assets/38147717/75a6cd65-db29-4aba-85b3-8cb03ec53929)

**Reply by charlie-ttt** [reply]
Hi @amyegan  sorry to ping you again. If you could spare a few minutes, could you advise, or connect me to anyone who may be able to help me?  I read a lot of online documentation on Vercel but still couldn't get it working - now I don't know how to proceed 😅

**Reply by amyegan** [reply]
Hi @charlie-ttt.  It looks like `kopiloti.com` is not currently using Vercel nameservers, IP address, or alias domain. I wouldn't expect the domain to show in your team or account domains list in this case.
- [kopilot.com NS](https://www.whatsmydns.net/#NS/kopiloti.com)
- [kopilot.com. A](https://www.whatsmydns.net/#A/kopiloti.com) 
- [www.kopilot.com CNAME](https://www.whatsmydns.net/#CNAME/www.kopiloti.com)

**Reply by charlie-ttt** [reply]
@amyegan Yes I had to switched it back since my company need access to their production google workspace email etc (while I cannot set up the DNS on vercel after the Nameservers switch). 

I just set the nameservers method like before and everything was propagated, although I'm still facing the same problem of not seeing the `enable DNS` in Vercel dashboard, and cannot manage DNS for this domain. (note: because of this, I have to switched back to GoDaddy again)

![image](https://github.com/vercel/community/assets/38147717/82a9d23e-874d-42ed-a753-08d37b9a55ac)

Could you let me know if you need other information that can be helpful? Really appreciate your time.

Not sure if this helps but on the Domain Page I have a bunch of 403 request on network tab

![image](https://github.com/vercel/community/assets/38147717/e970810e-2e6b-492b-b120-0c31724892e6)
